### ISSUE TITLE
Documentation: implementing the style for the <kbd> tag to style documentation instructions that refer to buttons

### ISSUE THREAD

**Issue by ogsamovar** [PrimeDAO/prime-deals-dapp#975]
The `<kbd>` element ([ref](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/kbd)) allows us to implement a dedicated styling for tasks/procedure that describes how to accomplish a task by clicking a certain button. 

The clarity and readability will improve if we style that part of the instructions in a distinctive way.

e.g.  consider the text:
_Please note that you can always come back to edit your input before you click the SUBMIT button on the last step of the wizard._

Now it simply shows like this:
<img width="1358" alt="Screen Shot 2022-05-09 at 21 25 26" src="https://user-images.githubusercontent.com/99011533/167482854-215f66c4-5943-411b-98de-5c9d81b2d82f.png">

but it can be showed in a more distinct manner, like this: 

<img width="1456" alt="Screen Shot 2022-05-09 at 21 08 31" src="https://user-images.githubusercontent.com/99011533/167480771-8deb8be9-ad20-4336-93fc-26f6011326fe.png">

if we style the `<kbd>` this way:

```
kbd {
    background-color: #eee;
    border-radius: 3px;
    border: 1px solid #b4b4b4;
    box-shadow: 0 1px 1px rgba(0, 0, 0, .2), 0 2px 0 0 rgba(255, 255, 255, .7) inset;
    color: #333;
    display: inline-block;
    font-size: .85em;
    font-weight: 700;
    line-height: 1;
    padding: 2px 4px;
    white-space: nowrap;
   }
```

Alternatively, we can render the `<kbd>` to look this way:
<img width="1302" alt="Screen Shot 2022-05-09 at 21 23 02" src="https://user-images.githubusercontent.com/99011533/167482514-d5d72793-4cc8-402a-b53f-94172b5b67cc.png">

if we style it this way:

```
kbd  {
    all: initial;
    outline: none;
    padding: 11px 13px;
    font-family: Inter;
    font-size: -2px;
    font-style: normal;
    font-weight: 700;
    line-height: 0px;
    letter-spacing: 0.6px;
    text-align: center;
    text-transform: uppercase;
    border-radius: 50px;
    cursor: pointer;
    background-color: rgba(0,0,0,0);
    transform: translateY(0);
    user-select: none;
    color: #f9f6f9;
    white-space: nowrap;
    gap: 12px;
    display: inline-flex;
    justify-content: center;
    align-items: center;
    transition: background .25s ease-in,transform .1s ease-in;
    border: none;
    background-image: linear-gradient(45deg, #a258a7 0%, #ff497a 50%, #8668fc 100%);
    background-size: 200% auto;
    color: #f9f6f9;
}
```

**Comment by dkent600** [comment]
I don't think we should make this look too much like a real live button.  Having said that, it might be possible to create a real live button for these, for what it is worth.  But not now, is too late.  @BartuCulha  @ogsamovar

**Comment by ogsamovar** [comment]
If you don't feel like mocking the real button we can keep it as a simple, more traditional <kbd> element by using these:


```
kbd {
    background-color: #eee;
    border-radius: 68px;
    border: 1px solid #b4b4b4;
    box-shadow: 0 1px 1px rgba(0, 0, 0, .2), 0 2px 0 0 rgba(255, 255, 255, .7) inset;
    color: #333;
    display: inline-block;
    font-size: .85em;
    font-weight: 600;
    line-height: 1;
    padding: 2px 5px;
    white-space: nowrap;
}
```

**Comment by ogsamovar** [comment]
@dkent600  as per your request on Telegram

`background-color: #F9F6F9;`
`border: 1px solid #DCDAE5;`    

~~box-shadow: 0 1px 1px rgba(0, 0, 0, .2), 0 2px 0 0 rgba(255, 255, 255, .7) inset;~~


`    color: #15062d;`

**Comment by BartuCulha** [comment]
Can we change the padding to this?
.markdownContainer kbd {
    padding: 3px 5px 2px 5px;
}

Currently:
It seems like to font we are using has 1px gap at its bottom. Therefore this objects looks unaligned.

![Vote-on-a-Deal-Documentation-Documentation-Prime-Deals (5)](https://user-images.githubusercontent.com/24368665/168068543-ce04190c-28aa-43df-85ea-abb19e4d61a4.png)


Expected:
With the updated padding, it will look like this.
![Vote-on-a-Deal-Documentation-Documentation-Prime-Deals (4)](https://user-images.githubusercontent.com/24368665/168068420-641d0624-9ec8-4b18-a273-d8957dd91a9c.png)

**Comment by dkent600** [comment]
done

**Comment by dkent600** [comment]
Hmm, actually didn't do. The given padding numbers did not do what was in the screenshot.  I changed to 3px 6px;

**Comment by dkent600** [comment]
And then I changed to this:

![image](https://user-images.githubusercontent.com/1821666/168170447-3a081dc1-7158-44b5-b68a-fb49d26c82df.png)
